tests/fixtures/parsers/space-before-function-paren/function-type-annotation.js

```diff
@@ -0,0 +1,525 @@
+"use strict";
```

Comment:
Good idea, I moved the fixture. I agree with reorganizing the folder in a separate PR.